operator/pkg/manifest/client.go

```diff
@@ -206,7 +206,7 @@ func (h *HelmClient) GetNsResource(actionClient *action.Install, operationType H
 	if err != nil {
 		return nil, err
 	}
-	return h.kubeClient.Build(bytes.NewBuffer(nsBuf), true)
+	return h.kubeClient.Build(bytes.NewBuffer(nsBuf), false)
```

Comment:
Is there a reason to set it to `false`?

Reply:
this will reduce the memory spent to validate the manifests by not having to introduce an OpenAPI Rendering. I assume these manifests are already prevalidated so it could be set to false. Any objections there?

Reply:
The manifest resources are not pre-validated, not that I know of. But since rendered manifests will be cached it will be a one time overhead?

Reply:
but anyway if it's an invalid manifest, the apiserver will rejected, right? Then the client side validation is kind of redundant imo.

Reply:
well, we have to differentiate between the build and installation process. But if pre-validation is not intended during build, I will approve the PR and we can discuss on a decision as a follow-up.